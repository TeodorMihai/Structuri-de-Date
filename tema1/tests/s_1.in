2 8
::w 3 90 ABCD
800
::w 0 1 AAAA
10
::r 3 90
30
::w 6 0 0002
3000
::d 2 1
1
::r 6 0
2
::d 6 0
3
::d 6 0
20
::w 1 31 BBBB
3
::d 3 90
4
::r 1 31
2
::w 0 0 BACD
600
::w 5 5 1111
8
::d 6 0
2
::w 0 0 AAAA
1
::w 6 8 FFFF
265
::d 6 2
30
::r 0 0 
1
::d 3 0
800
::e
