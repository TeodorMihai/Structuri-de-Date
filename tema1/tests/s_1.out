ABCD
33
36
0002
0000
748
1
13
BACD
3.58 0.44 1.17 0.47
